### DISCUSSION TITLE
Welcome to CtrlppCheck Discussions!

### DISCUSSION THREAD

**Discussion by dhoegerlETM** [siemens/CtrlppCheck#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cbv08** [comment]
Thank you for adding this discussions area!
I am Olav, I have recently started working with WinCC OA.
I have been tasked with getting a linter up and running. 

I am normally not one to post much on forums or social media. But I have quickly realized that my normal problemsolving approach, of finding someone else who had the same problem as me, is a lot harder with WinCC OA.

**Reply by dhoegerlETM** [reply]
Hello Olav! Well, Simatic WinCC OA is an expert system and needs thorough training and application expertise. You should not compare it to commonly known programming languages or tools on the internet.

**Reply by cbv08** [reply]
Hello Daniel.
That's fair, but even experts sometimes run into problems they don't know how to solve. 
Kind regards 
Olav
